public/data/abstraction.json

```diff
@@ -9,13 +9,23 @@
       "components": [
         "marker-list",
         "layers-graph"
-      ]
+      ],
+      "buttons": []
```

Comment:
What do you think of calling this 'actions'? It's a bit less vague than 'buttons'? If we decide on that, let's also rename the `step-buttons` accordingly.

Reply:
Good one 👍🏼